### ISSUE TITLE
Idea for new Feature: post to multiple spaces

### ISSUE THREAD

**Issue by simon-roderus** [humhub/humhub#1837]
It would be nice to be able to easily post something to multiple spaces without having to copy & paste the message.

**Comment by buddha87** [comment]
Can you give me an example of use for this feature?

**Comment by simon-roderus** [comment]
Sure! Example: I have spaces for different departments like "media", "programming" and "elearning" and I want to post a message that is relevant to those three spaces, but not to other spaces.

**Comment by buddha87** [comment]
Thanks, this could be implemented as extra module, I think this should not be part of the core, perhaps something like the breaking news module with a space filter or something would also be conceivable.

**Comment by 001101** [comment]
yes that would be nice, but it should be a separate module for the space and integrated in the post editor like it is in the mail module with the recipient address bar

**Comment by funkycram** [comment]
Some news ?

**Comment by dmgarland** [comment]
This seems like essential functionality to me. Suppose you have two groups that have some mutually exclusive members, but you wanted to post a photo to both groups. You'd need to upload twice and send to each group.

**Comment by freakms** [comment]
I'd like to push it again. This could be good. Now every user must be member of a special "infospace". It would be better to shout something into multiple spaces.

**Comment by Ace2213** [comment]
One way to implement this would be by adding a space selector to the post form in the dashboard. This way, the user can choose whether their post goes to their profile or to spaces, with checkboxes for the spaces.

I believe one fork of the share module had this kind of space picker, that lets the user choose multiple spaces. Here's my post where I discuss it: https://community.humhub.com/content/perma?id=199363

**Comment by tomofu74** [comment]
I'd like to push it, too. How about "copy" (not "move") the post?

**Comment by buddh4** [comment]
Wouldn't a global information sidebar snippet with important information, perhaps directed to a user group or specific space members. Users would have to read the information in order to remove it from the snippet, so you'd force users to read those informations.

**Comment by tomofu74** [comment]
Oh, global information sidebar snippet is good idea! Well, that is displayed on user's dashboard, but the spaces don't have  making method of "global". Our Administrator wants to display the information for space readers (including guest, anonymous) automatically when users make their own space. Is it capable?

**Comment by marc-farre** [comment]
The [Share content module](https://marketplace.humhub.com/module/sharebetween/description) should solve this issue I think.
